crux-kafka/src/crux/kafka.clj

```diff
@@ -265,7 +265,14 @@
 
   db/DocumentStore
   (submit-docs [this id-and-docs]
-    (submit-docs id-and-docs this))
+    (submit-docs id-and-docs this)
+
+    ;; for #1256 - let's fast-track tx-fn doc replacements straight into the local doc-store
```

Comment:
You kind of push the reader towards checking out the issue #1256 for explanation of this code path, but there's no actual description on that issue as to what the problem is, just a broken test.

Reply:
Good shout - reckon it's best to update the issue, the comment, or both?

Reply:
Gone for both :smile: